crates/interpreter/src/instructions/host_env.rs

```diff
@@ -54,3 +57,15 @@ pub fn origin(interpreter: &mut Interpreter, host: &mut dyn Host) {
     gas!(interpreter, gas::BASE);
     push_b256!(interpreter, host.env().tx.caller.into());
 }
+
+// EIP-4844: Shard Blob Transactions
+pub fn blob_hash<SPEC: Spec>(interpreter: &mut Interpreter, host: &mut dyn Host) {
```

Comment:
Having trouble understanding why `EIPTests/StateTests/stEIP4844-blobtransactions` are all failing, the implementation of the opcode is pretty simple. Might've missed setting up the env?

Reply:
If `max_fee_per_blob_gas` is set, I assume you need to do some gas calculation and determent the fee that is going to be deduced from the sender.
https://eips.ethereum.org/EIPS/eip-4844